frontend/src/metabase/admin/settings/components/widgets/EmbeddingSwitchWidget/EmbeddingSwitchWidget.tsx

```diff
@@ -0,0 +1,34 @@
+import { t } from "ttag";
+import * as MetabaseAnalytics from "metabase/lib/analytics";
+import { Stack, Switch, Text } from "metabase/ui";
+
+interface EmbeddingSwitchWidgetProps {
+  setting: { value: boolean | null };
+  onChange: (value: boolean) => void;
+}
+
+export const EmbeddingSwitchWidget = ({
+  setting,
+  onChange,
+}: EmbeddingSwitchWidgetProps) => (
+  <Stack spacing={"md"} className="text-measure">
+    <Text lh={1.5}>
+      {t`Embed dashboards, questions, or the entire Metabase app into your application. Integrate with your server code to create a secure environment, limited to specific users or organizations.`}
+    </Text>
+    <Switch
+      labelPosition="left"
+      checked={setting.value ?? false}
```

Comment:
It's more natural to cast this to boolean rather than specify a fallback value especially when the absence of the value means false.

```suggestion
      checked={Boolean(setting.value)}
```